### ISSUE TITLE
Issue with install

### ISSUE THREAD

**Issue by ghost** [svenschneider/youbot-manipulation#21]
Hello,

For reference, I am using Ubuntu 14.04 and ROS Indigo.

I am trying to follow your install, and this is what I have done so far:

1)  Create new catkin_ws by following the link you provided

2)  cd <my catkin workspace>/src

3)  catkin_ws/src$ wstool init   //this line was from another issue posted because I had an error without it

4) wstool set youbot_manipulation --git git@github.com:svenschneider/youbot-manipulation.git --version=indigo

5) wstool update youbot_manipulation  //This is where I got the error below

After step 5) I received this error:
cody@cody-Aspire-M5-582PT:~/catkin_ws/src$ wstool update youbot_manipulation
[youbot_manipulation] Fetching git@github.com:svenschneider/youbot-manipulation.git (version indigo) to /home/cody/catkin_ws/src/youbot_manipulation
Cloning into '/home/cody/catkin_ws/src/youbot_manipulation'...
Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
Exception caught during install: Error processing 'youbot_manipulation' : [youbot_manipulation] Checkout of git@github.com:svenschneider/youbot-manipulation.git version indigo into /home/cody/catkin_ws/src/youbot_manipulation failed.

ERROR in config: Error processing 'youbot_manipulation' : [youbot_manipulation] Checkout of git@github.com:svenschneider/youbot-manipulation.git version indigo into /home/cody/catkin_ws/src/youbot_manipulation failed.

cody@cody-Aspire-M5-582PT:~/catkin_ws/src$ 

Do you have any idea why I am getting this error?  

Thanks,
Cody

**Comment by shivangg** [comment]
You are using SSH to clone the repo the keys of which I'm afraid you have not configured correctly. Use the alternative method of cloning using https instead of SSH.

**Comment by Ahmed-Nouman** [comment]
You should use.
git clone http:...... Link
Instead of
git clone SSH.... Link

On Feb 22, 2017 1:44 PM, "Shivang Tripathi" <notifications@github.com>
wrote:

You are using SSH to clone the repo the keys of which I'm afraid you have
not configured correctly. Use the alternative method of cloning using https
instead of SSH.

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/svenschneider/youbot-manipulation/issues/21#issuecomment-281647099>,
or mute the thread

**Comment by arafatsh** [comment]
do you guys know how to install it for kinetic when i try catkin_make it says error:shared_ptr in namespace std does not name a template type
??
